django/db/models/functions.py

```diff
@@ -102,9 +102,12 @@ def __init__(self, *expressions, **extra):
             raise ValueError('Greatest must take at least two expressions')
         super(Greatest, self).__init__(*expressions, **extra)
 
-    def as_sqlite(self, compiler, connection):
+    def as_sqlite(self, compiler, connection, **kwargs):
         """Use the MAX function on SQLite."""
-        return super(Greatest, self).as_sql(compiler, connection, function='MAX')
```

Comment:
This line doesn't need to change. Passing `function='Max'` as a kwarg will do the right thing, since the super implementation will do the mixing of `self.extra` and `**kwargs` for you.